src/App/WorkerStats/WorkerStats.js

```diff
@@ -1,29 +1,35 @@
 import React from 'react';
-import { workerStats } from '../../../test/workerStats'
+import styles from './WorkerStats.css';
+import MinerInfo from '../MinerInfo/MinerInfo';
 
 export const WorkerStats = () => {
-    const { mature, immature, totalHash, totalShares, paid } = workerStats;
-    const miner = workerStats.miner.split('=');
-    
+    const minerData= {
```

Comment:
Im thinking we should probably start adding in TODOs for the content we need to replace:

e.g.
 /* TODO - replacing placeholder data */

Thoughts?